src/paradigm/views/greek/verb/greek-verb-paradigm-view.js

```diff
@@ -161,8 +161,14 @@ export default class GreekVerbParadigmView extends GreekView {
         let fullMatch = true
 
         for (const feature of comparativeFeatures) {
-          if (inflection.hasOwnProperty(feature)) {
-            fullMatch = fullMatch && cell[feature].hasValues(inflection[feature].values)
+          if (inflection.hasOwnProperty(feature) || (feature === 'lemma')) {
+
+            if (feature === 'lemma') {
+              fullMatch = fullMatch && inflection.word && cell[feature] === inflection.word.value
```

Comment:
maybe we should use the greek language model compareWords method here, instead of string ===

Reply:
Agree - it would be better! Fixed.